.github/workflows/deploy-to-environment.yml

```diff
@@ -13,22 +13,23 @@ on:
         options:
           - staging
           - beta
-        default: staging
       canister:
         required: true
         type: choice
         description: 'Select the canister to deploy.'
         options:
           - frontend
           - backend
-        default: frontend
       force-backend:
         required: false
         type: boolean
         description: 'Force backend deployment even if no changes are detected.'
         default: false
 
-run-name: Deploying ${{ inputs.canister }} to ${{ inputs.network }}
+run-name: >-
+  ${{ github.event_name == 'push'
+    && 'Deploying Frontend / Backend to Staging'
+    || format('Deploying {0} to {1} and force backend = {2}', inputs.canister, inputs.network, inputs.force-backend) }}
```

Comment:
can you make it so the "force backend" part appears only when the deploy is to Backend AND the force is true? is it possible?

Reply:
what if we add a 3rd option to the canister choice so we have:
- frontend
- backend
- both

with that the force-backend becomes obsolete. Or am I missing a case? Since chosing backend and force-backend is the exact same thing.

Reply:
yeah you are right. I think its possible but it might just result in an ugly string. Let me see what I can do